Assets/Scripts/OrchestraArmy/AudioManager.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.ComponentModel;
+using System.Linq;
+using OrchestraArmy.Entity.Entities.Players.WeaponSelection.Weapon.Weapons.Factory;
+using OrchestraArmy.Enum;
+using OrchestraArmy.Event;
+using OrchestraArmy.Event.Events.Player;
+using UnityEngine;
+
+namespace OrchestraArmy
+{
+    [Serializable]
+    public struct InstrumentAudio
+    {
+        public WeaponType Instrument;
+        public AudioClip C;
+        public AudioClip D;
+        public AudioClip E;
+        public AudioClip F;
+        public AudioClip G;
+        public AudioClip A;
+        public AudioClip B;
```

Comment:
Deze twee aub boven public AudioClip A zetten

Reply:
Indien we van A->B->C->D->... gaan ofc

Reply:
Alfabetisch is dat logisch, muzikaal wat minder. Noten lopen van C => D => E => F => G => A => B volgens de IPN. Een A binnen dezelfde pitch class zal dus altijd hoger zijn dan een C, vandaar dat ik automatisch die volgorde aanhoud ;)

Maar als je het liever als A => B => C => D => E => F => G ziet kan ik het aanpassen

Reply:
Als het logisch is, dan is dit prima :)